swiss-ng-api/berlin-group-documentation/current.yaml

```diff
@@ -6636,6 +6745,28 @@ components:
         The chosen XML schema of the status request is following the XML schema definitions of the original pain.001 schema.
       type: string
 
+    paymentInitiationStatusResponse-202_json:
```

Comment:
Relevant Change: new json response definition
Adoption approach: Follow the change